### PR TITLE
Add README and graphics

### PR DESCRIPTION
**Pull request by kev-ngo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,125 @@
...
+
+3. Run the `deploy-safe.ts` script to deploy the Safe smart account.
+
+```
+yarn deploy
+```
+
+> For the purposes of this demo, we're deploying to the Goerli test network. You can view your Safe on [Goerli Etherscan block explorer](https://goerli.etherscan.io/).
+
+4. Create a `.env` file at the root of the repo and add in the Safe address:
```
Comment (cbrzn) [inline README.md]: we need to put the creation of `.env` as first step, being the main reason that the user needs to define a private key, which will take care of paying for the deployment of the safe

File: README.md [hunk 2]
```diff
@@ -1,3 +1,125 @@
...
+
+2. Update the salt nonce in `deploy-safe.ts` to `0x` followed by a few randomly chosen digits. For example,
+
+This salt nonce is used later to create the new Safe address.
+
+```ts
+const SALT_NONCE = "0x185593";
+```
+
+3. Run the `deploy-safe.ts` script to deploy the Safe smart account.
```
Comment (cbrzn) [inline README.md]:
```suggestion
3. Run the `./scripts/deploy-safe.ts` script to deploy the Safe smart account.
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,125 @@
...
+
+First, you'll set up the demo.
+
+1. Change directory to `js`:
+
+```
+cd js
+```
+
+2. Update the salt nonce in `deploy-safe.ts` to `0x` followed by a few randomly chosen digits. For example,
```
Comment (cbrzn) [inline README.md]:
```suggestion
2. Update the salt nonce in `./scripts/deploy-safe.ts` to `0x` followed by a few randomly chosen digits. For example,
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,125 @@
...
+
+You should have received the Safe address after deploying it.
+
+Copy and paste this address into your `.env` file:
+
+```bash
+SAFE_ADDRESS="0x..."
+```
+
+5. Finally, add the a Safe transaction signer by running:
```
Comment (cbrzn) [inline README.md]: its worth mentioning that this is needed if you'd like to have **two** signers. when you deploy a safe, the owners are [defined in the arguments of the invocation](https://github.com/cbrzn/safe-playground/blob/main/js/scripts/deploy-safe.ts#L24)

Comment (cbrzn) [reply README.md]: Also, here we need to specify that the user should modify the [`OWNER` variable](https://github.com/cbrzn/safe-playground/blob/main/js/scripts/owners/add.ts#L4) with the desired address

File: README.md [hunk 5]
```diff
@@ -1,3 +1,125 @@
...
+
+## Multi-platform Support: Rust
+
+One of the powerful advantages of the AA Wrap is that it can be used in any app, as long as the app has a Polywrap client library installed.
+
+The above scripts were run in a JavaScript environment. We've also prepared scripts that run in **Rust** below.
+
+First, you need to be in the `rs` directory.
+
+Then, update the salt nonce in the `deploy.rs` file:
```
Comment (cbrzn) [inline README.md]:
```suggestion
Then, update the salt nonce in the `./bin/deploy.rs` file:
```

File: README.md [hunk 6]
```diff
@@ -1,3 +1,125 @@
...
+cargo run --bin --release <script>
+```
+
+Where `<script>` is one of the following:
+
+| Script       | Description                     |
+| ------------ | ------------------------------- |
+| `deploy`     | Deploys your Safe smart account |
+| `add_owners` | Add owners to the Safe          |
+| `get_owners` | Get the list of owners          |
```
Comment (cbrzn) [inline README.md]: we can also add sponsor transaction https://github.com/cbrzn/safe-playground/blob/main/rs/src/bin/sponsor_transaction.rs

**Review by cbrzn — COMMENTED** [review]
this is awesome ser :smiling_face_with_tear: left a few comments which should be addressed before merging :smile:

**Comment by kev-ngo** [comment]
See new PR

**Pull request closed without merging**